samples/dashboards/data/mqtt-connector/demo.js

```diff
@@ -0,0 +1,645 @@
+/* *
+ *
+ *  Sample application using a MQTT connector (custom connector)
+ *
+ * */
+
+// Global board instance for use in event handlers
+let board = null;
+
+
+// Highcharts common options
+Highcharts.setOptions({
```

Comment:
This is absolutely correct, but: since we use spread operators for the other config objects then perhaps we should do it here as well? Just `commonChartOptions` as a constant and then `chartOptions: ...commonChartOptions` further down? Just to keep the code cleaner and more consistent across, and not introduce "unnecessary" methods even though "correct".

Controversial, so feel free to ignore this comment ;-)

Reply:
A copy-paste from some other project, I'll rewrite using the spread opearator for consistency.